source/shared/cpp/ObjectModel/TextElementProperties.cpp

```diff
@@ -167,9 +167,14 @@ void TextElementProperties::SetLanguage(const std::string& value)
     m_language = value;
 }
 
-void TextElementProperties::Deserialize(const ParseContext& context, const Json::Value& json)
+void TextElementProperties::Deserialize(ParseContext& context, const Json::Value& json)
 {
-    SetText(ParseUtil::GetString(json, AdaptiveCardSchemaKey::Text, true));
+    SetText(ParseUtil::GetString(json, AdaptiveCardSchemaKey::Text, false));
```

Comment:
>GetString [](start = 23, length = 9)

We actually want this behavior for all required properties going forward now, right (#4565)? Rather than adding this code here specifically in TextBlock, do we want to update GetString to not throw and to instead add a warning when a required property is missing? That probably requires some follow up in the renderers to make sure that we don't crash on missing properties. But otherwise we'll end up with all "required" properties having this block of code and never passing `true` to the ParseUtil functions.

Reply:
As discussed, I've created a new issue to track to update all of the required string parsing as part of [https://github.com/microsoft/AdaptiveCards/issues/4565](https://github.com/microsoft/AdaptiveCards/issues/4565).  [https://github.com/microsoft/AdaptiveCards/issues/4834](https://github.com/microsoft/AdaptiveCards/issues/4834) is the one track this particular issue.

---
In reply to: [494592127](https://github.com/microsoft/AdaptiveCards/pull/4765#discussion_r494592127) [](ancestors = 494592127)